packages/salesforcedx-vscode-core/src/conflict/metadataCacheService.ts

```diff
@@ -0,0 +1,273 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
+  ComponentSet,
+  MetadataApiRetrieve,
+  RetrieveResult,
+  SourceComponent,
+  SourceRetrieveResult
+} from '@salesforce/source-deploy-retrieve';
+import { FileProperties } from '@salesforce/source-deploy-retrieve/lib/src/client/types';
```

Comment:
we should make a note to export `FileProperties` at the top level in SDR

Reply:
Let's file a WI and put it in the epic. Even if it's gets put below the line.

Reply:
W-9225729